portal/src/components/common/BaseImagePicker.tsx

```diff
@@ -91,6 +144,26 @@ const BaseImagePicker: React.VFC<BaseImagePickerProps> =
           accept="image/png, image/jpeg, image/gif"
           onChange={onInputChange}
         />
+        <Dialog
+          hidden={isSizeLimitErrorDialogHidden}
+          onDismiss={onDialogDismiss}
+          dialogContentProps={dialogContentProps}
+        >
+          <DialogFooter>
+            <PrimaryButton
+              onClick={onRetry}
+              text={renderToString(
+                "BaseImagePicker.size-dialog.button-retry-label"
+              )}
+            />
+            <DefaultButton
+              onClick={onDialogDismiss}
+              text={renderToString(
+                "BaseImagePicker.size-dialog.button-cancel-label"
+              )}
+            />
+          </DialogFooter>
+        </Dialog>
```

Comment:
There is a `ImageInput` component in the v2 ui, do we need to update it so the ux is the same? (But it is not fluent style 🤔 )

Reply:
`ImageInput` has handled size limit already. But the size limit prop is not mandatory.

I am going to rename the prop to `sizeLimitInBytes` and make it required. So all these image picker components have the same prop dealing with size limit.

However, but the error handling is not trivial to port. `ImageInput` is v2 ui and it is backed by `FormField`, an abstraction that have native support for field-level error.